Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -1082,6 +1082,38 @@ StringView gstStructureGetName(const GstStructure* structure)
     return StringView::fromLatin1(gst_structure_get_name(structure));
 }
 
+template<typename T>
+Vector<T> gstStructureGetArray(const GstStructure* structure, ASCIILiteral key)
```

Comment:
Would be nice to have unit tests for this, see Tools/TestWebKitAPI/Tests/WebCore/gstreamer/GStreamerTest.cpp

Reply:
I'll create a seperate bug for that. This task has been on for 3 months already and I wouldn't like it to delay more its landing now, if possible.

Reply:
Follow up bug created for this unit test writing task: https://bugs.webkit.org/show_bug.cgi?id=280190